docs/source/en/serialization.mdx

```diff
@@ -491,9 +491,6 @@ Exporting a model requires two things:
 
 These necessities imply several things developers should be careful about. These are detailed below.
 
-
-### Implications
```

Comment:
This is redefined later in that file and that section is empty here, so I assume it's by mistake (cc @lewtun )

Reply:
Thanks for the fix!